src/__tests__/testid-test.tsx

```diff
@@ -101,6 +109,15 @@ test('Cards test ids', () => {
     );
 });
 
+test('HighlightedCard test ids', () => {
+    checkTestIds(<HighlightedCard title="Title" description="Description" imageUrl="https://anyurl.com" />, [
```

Comment:
is the spec updated? I can't see "image" in specs
https://www.figma.com/design/tKdPOfcUALzVIh5oizFbm7/%F0%9F%94%B8-Cards-Specs?node-id=9583-1016&t=UQPGAgvhx5J6ErMZ-4

Reply:
nop, I didn't change the cards specs, I'll open a branch for this

Reply:
https://www.figma.com/design/oaSYsDNA70XjZ3KOLdvp4T/testids-highlighed-card?node-id=9583-1016&t=xtM8CYBGQENVsd46-4